player.sql

```diff
@@ -0,0 +1,10 @@
+CREATE DATABASE playerlist;
```

Comment:
Is it possible to put `player.sql`, `db.js`, and `index.js` into a server folder to clean things up?